src/lib.rs

```diff
@@ -99,6 +99,8 @@ macro_rules! define_if_single_runtime_enabled {
 // In the case that neither tokio nor async-std is enabled, we want to disable all compiler errors
 // and warnings other than our custom ones.
 define_if_single_runtime_enabled! {
+    #[macro_use]
+    pub mod error;
```

Comment:
Rather than `#[macro_use]`, we can rely on the more "modern" `pub(crate) use <macro name>;` inside of the `error` module. See https://stackoverflow.com/a/67140319

Reply:
Ah, didn't know about that, much nicer - done.